src/components/ContextMenu/ContextMenu.less

```diff
@@ -0,0 +1,7 @@
+.lucid-ContextMenu {
+	display: inline-block;
+}
+
+.lucid-ContextMenu-FlyOut {
```

Comment:
Oy, I haven't been using the Capital name convention for children class names. I'll adjust my style accordingly.